reliability/reliability.md

```diff
@@ -28,4 +28,6 @@ Uptime monitoring is traditionally done at the server level or by checking one o
 While a site's services may be responding, to a user, a site being "up" means more than this to them. Performance monitoring is similar to uptime monitoring, but also takes note of certain metrics that could indicate trouble. Metrics like "page load time" and "slowest average transactions" should be monitored and reported regularly to help keep you ahead of performance issues. Monitoring slow logs for problematic queries or requests can also help keep user sites stable. MySQL, PHP-FPM, and others provide options to capture these for monitoring.
 
 ## Version Control
-Version control allows a user or host to store, view, or revert changes to individual components of a site over time. It's recommended to provide end users a way to manage their site over version control, as it encourages best practices and is a great extra safety net for their WordPress site.
+Version control is a way of tracking the changes made to files over time, especially for sites running WordPress for example. It allows users to track the revision history of code and to revert or apply changes easily via the command line. It is also a good way to debug your website if something goes wrong as you can quickly restore to a previous version very quickly without doing a full backup. 
```

Comment:
> Version control is a way of tracking the changes made to files over time, especially for sites running WordPress for example.

The end of this doesn't quite sound right to me. Version control is great for all kinds of things, beyond websites and WordPress. Maybe we can mention it is a best practice when working with files, such as code, that change over time.

Reply:
> if something goes wrong as you can quickly restore to a previous version very quickly without doing a full backup. 

I interpret "doing a full backup" as creating a backup, not restoring one.

Should this be something like "if something goes wrong as you can quickly restore to a previous state of the site code without restoring a full backup. "?